set.mm

```diff
@@ -19926,17 +19926,17 @@ theorem as an axiom of set theory (Axiom 0 of [Kunen] p. 10).  In the
     $d x ps $.
     spimv.1 $e |- ( x = y -> ( ph -> ps ) ) $.
     $( A version of ~ spim with a distinct variable requirement instead of a
-       bound variable hypothesis.  See also ~ spimv1 and ~ spimvw .  See also
-       ~ spimvALT .  (Contributed by NM, 31-Jul-1993.)  Removed dependency on
-       ~ ax-10 .  (Revised by BJ, 29-Nov-2020.) $)
+       bound variable hypothesis.  See also ~ spimv1 and ~ spimvw .
+       (Contributed by NM, 31-Jul-1993.) $)
     spimv $p |- ( A. x ph -> ps ) $=
-      ( weq wi ax6e eximii 19.36iv ) ABCCDFABGCCDHEIJ $.
+      ( nfv spim ) ABCDBCFEG $.
 
-    $( Alternate proof of ~ spimv .  Shorter but requires more axioms.
-       (Contributed by NM, 31-Jul-1993.)  (Proof modification is discouraged.)
+    $( Obsolete version of ~ spimv as of 8-Jul-2022.  (Contributed by NM,
+       31-Jul-1993.)  Removed dependency on ~ ax-10 .  (Revised by BJ,
+       29-Nov-2020.)  (Proof modification is discouraged.)
        (New usage is discouraged.) $)
-    spimvALT $p |- ( A. x ph -> ps ) $=
-      ( nfv spim ) ABCDBCFEG $.
+    spimvOLD $p |- ( A. x ph -> ps ) $=
```

Comment:
Can you keep it as ALT, with the comment:
> Alternate proof of  ~ spimv .  Note that it requires only ~ ax-1 -- ~ ax-5 together with ~ ax6e (so that it requires only ~ ax-1 -- ~ ax-6 if we allow use of ~ ax-6 ).

Thanks.

Reply:
spim (and likely some other theorems) can likewise be proven from ax-6 directly.

Reply:
The current proof of spimv does not show that it can be proved from ax-1--ax-6. If and when it becomes clear, then maybe we can name the theorem spimvOLD, but until then, I'd like to keep it as spimvALT.